### ISSUE TITLE
Feature Request: Allow to use TAB for choosing connection instead drowpdown

### ISSUE THREAD

**Issue by mdemori** [subhra74/snowflake#66]
Hello, 

will be awesome if it will be possible to select all current terminals using tabs list instead dropdown menu.
It will be faster for switching between server connection

**Comment by andreshans** [comment]
+1

**Comment by MartinCicmanec** [comment]
Yes, terminal tabs on top of terminal window and server connection tabs on left side. I would love that.

**Comment by rmfstar** [comment]
Ditto, mPutty does this very well, i know its only an extra click for the dropdown but really improves the experience having the open connections as tabs accross the top.

**Comment by Samahedron** [comment]
+1 Definitely this. When working with multiple sessions, it's nice to have an overview at a glance of what's open.
